scripts/githooks/commit_msg.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env python
```

Comment:
This will not work if your system python is not Python 2.x which is pretty common, ideally this script should work with both python 2.x and 3.x and not hardcode a specific major version

Reply:
AFAIK, the code should now be Python3 compliant.